test/integration/redo_undo/light/redo_undo_ambientLight_test.re

```diff
@@ -36,6 +36,12 @@ let _ =
 
       MainEditorSceneTool.initState(~sandbox, ());
 
+      LocalStorage.setValue(HeaderNotice.Method.getWelComeUserKey(), "ok");
```

Comment:
TODO remove this

Reply:
use  BuildComponentForCurryTool.buildController instead of  BuildComponentForCurryTool.buildHeader